src/types/schema/apisix/plugin-global-rule.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from 'zod';
+import { A6Common } from './common';
+import { A6Plugin } from './plugin';
+
+const PluginGlobalRule = z
```

Comment:
Why must this resource be named differently than Admin API?

Reply:
I guess @SkyeYoung is following https://docs.api7.ai/apisix/key-concepts/plugin-global-rules?

Reply:
yes.